charts/consul/test/unit/enterprise-license-job.bats

```diff
@@ -29,24 +29,6 @@ load _helpers
       .
 }
 
-@test "enterpriseLicense/Job: disabled when secretName is missing" {
```

Comment:
We have a hard failure in [server-statefulset](https://github.com/hashicorp/consul-k8s/pull/1032/files#diff-3b78b7152b597d177763b2ffdad9fab0bde0b6e1a0c1080b439a5fa7121ab794R1378-R1398) when only one of `global.enterpriseLicense.secretKey` or `global.enterpriseLicense.secretName` are provided.
These tests are redundant and were only soft failing by not including that configuration.  Now the overall helm chart install/upgrade/apply will fail if these conditions are true.